app/models/appt.rb

```diff
@@ -0,0 +1,3 @@
+class Appt < ApplicationRecord
```

Comment:
@Smokashi23 Use proper model name 
```suggestion
class Appointment < ApplicationRecord
```